### DISCUSSION TITLE
[@types/react] fix type error for strict mode

### DISCUSSION THREAD

**Discussion by long76** [DefinitelyTyped/DefinitelyTyped#62901] [Issues with a @types package]
replace `| undefined` to `| null | undefined`

example fix see: https://github.com/long76/DefinitelyTyped/commit/6d4b0582d031d4af45b0ffa0a2601aee1df99b2c

thanks!

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
Why?

**Reply by long76** [reply]
Because in strict mode undefined != null

**Reply by long76** [reply]
> Because in strict mode undefined != null

Amd i got compilation error. It's easy fix
